### ISSUE TITLE
Uptime command

### ISSUE THREAD

**Issue by rtk-rnjn** [python-discord/sir-lancebot#1015]
## Description
Uptime command
https://github.com/python-discord/sir-lancebot/blob/main/bot/exts/core/ping.py#L29-L40

## Reasoning
It look cool

## Proposed Implementation
Use Relative timestamp of discord `<t:TIMESTAMP:R>`

## Would you like to implement this yourself?

- [ ] I'd like to implement this feature myself
- [x] Anyone can implement this feature

**Comment by MaskDuck** [comment]
can i be assigned?

**Comment by TizzySaurus** [comment]
> can i be assigned?

This hasn't been approved yet, but if/when it does then I don't see an issue with that.

**Comment by Akarys42** [comment]
This command [already exists](https://github.com/python-discord/sir-lancebot/blob/main/bot/exts/core/ping.py#L29).

**Comment by TizzySaurus** [comment]
> This command [already exists](https://github.com/python-discord/sir-lancebot/blob/main/bot/exts/core/ping.py#L29).

They're suggesting that the command be updated to use relative timestamps instead of manually getting humanized time.

**Comment by Akarys42** [comment]
Right, now I understand 😅

**Comment by MaskDuck** [comment]
can i be assigned

**Comment by mbaruh** [comment]
Timestamps do rounding, which isn't something I want if I need to check for uptime. Relative timestamps also don't work on mobile.

**Comment by vcokltfre** [comment]
I agree with @mbaruh, the rounding on relative timestamps makes them annoying for things like this

**Comment by MaskDuck** [comment]
didnt know =(
